test/e2e/snaps/test-snap-management.spec.js

```diff
@@ -33,31 +33,24 @@ describe('Test Snap Management', function () {
         await driver.delayFirefox(1000);
 
         // wait for and click connect
-        await driver.waitForSelector('#connectnotifications');
```

Comment:
:information_source:  removing all the `waitForSelectors` before clicks as they are redundant steps